test/lit/passes/asyncify_pass-arg=asyncify-memory@memory.wast

```diff
@@ -0,0 +1,116 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt --enable-multimemory --enable-mutable-globals --asyncify --pass-arg=asyncify-memory@memory -S -o - | filecheck %s
+
+;; This test checks that the asyncify-memory argument can pick which memory to use in asyncify.
+(module
+  ;; CHECK:      (type $0 (func (param i32)))
+
+  ;; CHECK:      (type $1 (func))
+
+  ;; CHECK:      (type $2 (func (result i32)))
+
+  ;; CHECK:      (global $__asyncify_state (mut i32) (i32.const 0))
+
+  ;; CHECK:      (global $__asyncify_data (mut i32) (i32.const 0))
+
+  ;; CHECK:      (memory $unused0 1 1)
+  (memory $unused0 (export "unused0") 1 1)
+  ;; CHECK:      (memory $mem 1 1)
+  (memory $mem (export "memory") 1 1)
+  ;; CHECK:      (memory $ignore 1 1)
+  (memory $ignore (export "unused") 1 1)
+)
+;; CHECK:      (export "unused0" (memory $unused0))
+
+;; CHECK:      (export "memory" (memory $mem))
+
+;; CHECK:      (export "unused" (memory $ignore))
+
+;; CHECK:      (export "asyncify_start_unwind" (func $asyncify_start_unwind))
+
+;; CHECK:      (export "asyncify_stop_unwind" (func $asyncify_stop_unwind))
+
+;; CHECK:      (export "asyncify_start_rewind" (func $asyncify_start_rewind))
+
+;; CHECK:      (export "asyncify_stop_rewind" (func $asyncify_stop_rewind))
+
+;; CHECK:      (export "asyncify_get_state" (func $asyncify_get_state))
+
+;; CHECK:      (func $asyncify_start_unwind (param $0 i32)
+;; CHECK-NEXT:  (global.set $__asyncify_state
+;; CHECK-NEXT:   (i32.const 1)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT:  (global.set $__asyncify_data
+;; CHECK-NEXT:   (local.get $0)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT:  (if
+;; CHECK-NEXT:   (i32.gt_u
+;; CHECK-NEXT:    (i32.load $unused0
```

Comment:
This doesn't look right - it is reading from the memory which should not be used? It should be `$mem`.

Reply:
That was a build error on my part. fixed in 58e7cf